multispaeti/_multispati_pca.py

```diff
@@ -46,27 +52,24 @@ class MultispatiPCA:
         A distance matrix should be transformed to connectivities by e.g.
         calculating :math:`1-d/d_{max}` beforehand.
 
-    Raises
```

Comment:
why has this section been removed the error raise still occurs in __init__

Reply:
No, it doesn't. All checks are performed in fit to make it sklearn compatible